autorest/azure/persist.go

```diff
@@ -49,7 +49,7 @@ func SaveToken(path string, mode os.FileMode, token Token) error {
 
 	// Atomic replace to avoid multi-writer file corruptions
 	if err := os.Rename(tempPath, path); err != nil {
-		return fmt.Errorf("failed to move temporary token to desired output location. source=(%s). destination=(%s). error = %v", tempPath, path, err)
+		return fmt.Errorf("failed to move temporary token to desired output location. source=(%s). destination=(%s). error(rename)=%q", tempPath, path, err)
```

Comment:
the format can be better as:

    failed to move temporary token to desired output location: src=%q dsts=%q: %v